### PR TITLE
details in Zariski-Van-Kampen

### PR DESCRIPTION
**Pull request by fchapoton**
Small changes in the modified files, about pep8 and ruff warnings about code

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

**Comment by enriqueartal** [comment]
There is a related PR #36768 with more changes in this file and with changes in some other files. A merge?

File: src/sage/schemes/curves/zariski_vankampen.py [hunk 1]
```diff
@@ -847,7 +847,7 @@ def braid_in_segment(glist, x0, x1, precision={}):
         sage: B = braid_in_segment(glist, p1b, p2b); B              # optional - sirocco
         s5*s3^-1
     """
-    precision1 = {_: precision[_] for _ in precision.keys()}
+    precision1 = {k: v for k, v in precision.items()}
```
Comment (mantepse) [inline src/sage/schemes/curves/zariski_vankampen.py]: isn't that the same as `precision1 = precision.copy()` ?

File: src/sage/schemes/curves/zariski_vankampen.py [hunk 2]
```diff
@@ -1315,7 +1316,7 @@ def conjugate_positive_form(braid):
             A1 = rightnormalform(sg)
             par = A1[-1][0] % 2
             A1 = [B(a) for a in A1[:-1]]
-            if len(A1) == 0:
+            if not A1:
                 b = B.one()
             else:
                 b = prod(A1)
```
Comment (grhkm21) [inline src/sage/schemes/curves/zariski_vankampen.py:1322]: I believe you can make this `b = prod(A1, B.one())`.

File: src/sage/schemes/curves/zariski_vankampen.py [hunk 3]
```diff
@@ -1378,8 +1379,8 @@ def braid2rels(L):
     br1 = B0.delta()**r * B0(prod(B0(_) for _ in br0_left[1:]))
     cox = prod(F0.gens())
     U0 = [cox**q * (f0 * br1) / cox**q / f0 for f0 in F0.gens()[:-1]]
-    U = [tuple(sign(k1)*(abs(k1) + k) for k1 in _.Tietze()) for _ in U0]
-    pasos = [B.one()] + [_ for _ in reversed(L1)]
+    U = [tuple(sign(k1) * (abs(k1) + k) for k1 in _.Tietze()) for _ in U0]
+    pasos = [B.one()] + list(reversed(L1))
```
Comment (grhkm21) [inline src/sage/schemes/curves/zariski_vankampen.py:1383]:
```diff
- br1 = B0.delta()**r * B0(prod(B0(_) for _ in br0_left[1:]))
+ br1 = B0.delta()**r * prod(map(B0, br0_left[1:]), B0.one())
```

Also, do you mind renaming `_` to say `br` or something sensible? Variable names shouldn't start with underscores if they are used.

**Review by grhkm21 — APPROVED** [review]
Since your PR focuses on details / Pythonic changes, I suggested some too, but they are optional. Otherwise, looks good to me.

**Pull request merged**